Tests/ValidationTests.swift

```diff
@@ -360,10 +358,9 @@ class MultipleValidationTestCase: BaseTestCase {
         // Then
         XCTAssertNotNil(error)
 
-        if let error = error {
-            XCTAssertEqual(error.domain, ErrorDomain)
-            XCTAssertEqual(error.code, ErrorCode.statusCodeValidationFailed.rawValue)
-            XCTAssertEqual(error.userInfo[ErrorUserInfoKeys.StatusCode] as? Int, 200)
+        if let error = error as? AFError, let statusCode = error.responseCode {
+            XCTAssertTrue(error.isUnacceptableStatusCode)
+            XCTAssertEqual(statusCode, 200)
         } else {
             XCTFail("error should not be nil")
         }
```

Comment:
You can definitely compare optional and non-optional types using `XCTAssertEqual`. I do it all the time. I'll move this through for now and make a note to update it.